calblueprint/meepanyar
# Schema and Patch Regeneration

[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR
- Regenerated schema after adding the following columns:
  - Organization string column for Users (we don’t really need to interact with this AFAIK)… Will leave the NPO to change this to some linked record after handoff
  - Admin boolean for Users
  - “Grace Period” number column for Sites (the number of days of buffer that the user has to collect payments from the customers they metered in the previous period)
  - “Customer Number” number column for Sites (A number set by the User for each customer so that they can be telled apart from each other)
  - Changed column name "Approved At" to "Reviewed At" in the Purchase Request table 
  - Users table  rename "Purchase Requests" to "Purchase Requests Created"
  - Users table  rename  “Purchase Requests 2” to “Purchase Requests Reviewed”
  - Remove Period fields from the Inventory Updates and Purchase Requests tables
  - Inventory Updates table rename "updatedAt"  to "createdAt"

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

## How to review
- Regenerate the schema and `git status` to ensure both the request and schema files are modified
- Apply the patches via `npm run apply-schema-patches` and do `git status` and make sure the commit tree is clean
- Ensure all changes made to the schema are wanted and proper
- Run `npm start` to make sure no errors occur from changing interfaces

[//]: # 'The order in which to review files and what to expect when testing locally'

## Relevant Links

### Online sources

[//]: # 'Optional - copy links to any tutorial or documentation that was useful to you when working on this PR'

### Related PRs

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @julianrkung

[//]: # 'This tags Julian as a default. Feel free to change, or add on anyone who you should be in on the conversation.'
